lib/conversation/replies.js

```diff
@@ -100,7 +100,7 @@ function renderContinueConversationMsg(fn) {
             logger.verbose(`saved user:${args.req.user._id} current_campaign:${args.req.user.current_campaign}`);
             fn(args);
             args.req.user.postDashbotOutgoing(args.replyText);
```

Comment:
The Dashbot analytics should be tracking the messageTemplate and not the actual rendered message:
<img width="1032" alt="screen shot 2017-07-07 at 8 54 04 am" src="https://user-images.githubusercontent.com/1236811/27965877-049661ca-62f2-11e7-9499-bb7a7f5f6962.png">